e2e-tests/IIoTPlatform-E2E-Tests/TestModels/PublisherExtensions.cs

```diff
@@ -16,12 +16,12 @@ public static class PublisherExtensions {
         /// <summary>
         /// Create an api model
         /// </summary>
-        public static PublishNodesRequestApiModel ToApiModel(
+        public static PublishNodesEndpointApiModel ToApiModel(
             this PublishedNodesEntryModel model) {
             if (model == null) {
                 return null;
             }
-            return new PublishNodesRequestApiModel {
+            return new PublishNodesEndpointApiModel {
```

Comment:
Aren't we missing assignments of `DataSetWriterGroup` and `DataSetPublishingInterval` properties here ? Both classes seem to have them.